kubectl-plugin/pkg/cmd/create/create_workergroup.go

```diff
@@ -21,18 +21,19 @@ import (
 )
 
 type CreateWorkerGroupOptions struct {
-	configFlags       *genericclioptions.ConfigFlags
-	ioStreams         *genericclioptions.IOStreams
-	clusterName       string
-	groupName         string
-	rayVersion        string
-	image             string
-	workerCPU         string
-	workerGPU         string
-	workerMemory      string
-	workerReplicas    int32
-	workerMinReplicas int32
-	workerMaxReplicas int32
+	configFlags          *genericclioptions.ConfigFlags
+	ioStreams            *genericclioptions.IOStreams
+	workerRayStartParams map[string]string
```

Comment:
```suggestion
	rayStartParams map[string]string
```

Simply `rayStartParams` because it must be worker for `create workergroup`.

Reply:
makes sense, updated